src/PrestaShopBundle/Controller/Admin/Sell/Catalog/Product/CombinationController.php

```diff
@@ -348,23 +347,25 @@ public function getListAction(int $productId, ProductCombinationFilters $combina
     /**
      * @AdminSecurity("is_granted('update', request.get('_legacy_controller'))")
      *
-     * @param int $productId
```

Comment:
This param is still present it should be kept, or all must be removed

Reply:
idk why git shows it this way, its there, maybe outdated change.